tests/BaseRequestTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+use DDB\OpenPlatform\OpenPlatform;
+use DDB\OpenPlatform\Exceptions\InvalidPropertyException;
+use DDB\OpenPlatform\Request\BaseRequest;
+use DDB\OpenPlatform\Response\Response;
+use DDB\OpenPlatform\Response\SearchResponse;
+use PHPUnit\Framework\TestCase;
+
+// phpcs:ignore PSR1.Classes.ClassDeclaration.MissingNamespace
+class BaseRequestTest extends TestCase
+{
+    public function testSubclassesMustSetAPath()
+    {
+
```

Comment:
Any reason behind the additional whitespace at the beginning of each method here?